src/App/App.js

```diff
@@ -0,0 +1,75 @@
+import * as React from "react";
+import { useState, useCallback } from "react";
+
+import './styles.scss';
+
+function App() {
+  const [tiles, setTiles] = useState([]);
+  const [shuffledTiles, setShuffledTiles] = useState([]);
+  const [isTilesShuffled, setIsTilesShuffled] = useState(false);
+  const orderTiles = useCallback((async) => {
+    const order = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+    let tilesInProgress = [...tiles];
+    if (tiles.length === 0) {
+      for (let i = 0; i < order.length; i++) {
+        tilesInProgress.push(
+          <div className="drag-item" draggable="true" key={i}>
+            <p className="tile-number">{order[i]}</p>
+          </div>
+        )
+      }
+      tilesInProgress.push(
+        <div className="drag-item" draggable="true" id="empty" key="empty"></div>
+      )
+      setTiles(tilesInProgress);
+    } else {
+      setIsTilesShuffled(false);
+    }
+  }, [tiles]);
+
+  
+  const shuffleTiles = useCallback(() => {
+    let currentIndex = tiles.length,  randomIndex;
+    let tilesInProgress = [...tiles];
+    // While there remain elements to shuffle.
+    while (currentIndex > 0) {
+
+      // Pick a remaining element.
+      randomIndex = Math.floor(Math.random() * currentIndex);
+      currentIndex--;
+
+      // And swap it with the current element.
+      [tilesInProgress[currentIndex], tilesInProgress[randomIndex]] = [
+        tilesInProgress[randomIndex], tilesInProgress[currentIndex]];
```

Comment:
maybe make this one line or multiple lines? 👍 
`
[
        tilesInProgress[currentIndex], 
        tilesInProgress[randomIndex]
] = [
        tilesInProgress[randomIndex], 
        tilesInProgress[currentIndex]
];
`

not sure if that is possible in javascript, but it makes it more readable

Reply:
I will try to add and apply global formatting in the next PR